pager/src/main/java/com/google/accompanist/pager/Pager.kt

```diff
@@ -219,6 +219,13 @@ internal fun Pager(
 
     val pagerScope = remember(state) { PagerScopeImpl(state) }
 
+    // We only consume nested flings in the main-axis, allowing cross-axis flings to propagate
+    // as normal
+    val consumeFlingNestedScrollConnection = ConsumeFlingNestedScrollConnection(
```

Comment:
It's a class now, let's remember it as it is very important to remember the connection so we do less work reattaching it.